### PR TITLE
#855 Df component

### PR DESCRIPTION
**Pull request by SneakyTurtlle**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,8 @@
   },
   "devDependencies": {
     "@vitest/coverage-v8": "^0.33.0",
+    "eslint": "^8.52.0",
+    "eslint-plugin-unicorn": "^48.0.1",
```
Comment (adamPrestor) [inline package.json:26]: Why is this removed?

If this was intentional, please provide a reason and run `npm install` to update package-lock and push it to repo. This is the reason tests are failing.

File: vue/dynamicforms/src/index.ts [hunk 2]
```diff
@@ -54,6 +55,7 @@ export function createDynamicForms(options: DynamicFormsOptions = defaultOptions
 
       // import all global instances that we need for vuetify to work
       Object.entries(VuetifyComponents).map(([name, component]) => app.component(name, component));
+      app.component('df-app', DfApp);
```
Comment (adamPrestor) [inline vue/dynamicforms/src/index.ts:58]: This probably raises an error?

You can change 'df-app' to 'DfApp'. This should fix the error and preserve functionality.

**Review by adamPrestor — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (SneakyTurtlle) [reply package.json:26]: hello

File: vue/dynamicforms/src/index.ts (hunk 2 above)
Comment (adamPrestor) [inline vue/dynamicforms/src/index.ts:58]: I still get this error in the console:
~/vue/dynamicforms/src/index.ts
  58:21  warning  Property name "df-app" is not PascalCase  vue/component-definition-name-casing

Comment (adamPrestor) [reply vue/dynamicforms/src/index.ts:58]: To reproduce... npm run dev -> console output

**Review by adamPrestor — CHANGES_REQUESTED** [review]

File: vue/dynamicforms/src/index.ts (hunk 2 above)
Comment (SneakyTurtlle) [reply vue/dynamicforms/src/index.ts:58]: I already changed this once based on a comment, from dfApp -> df-app.

So should it be PascalCase or kebab-case?

**Review by adamPrestor — APPROVED** [review]

**Pull request merged**